### PR TITLE
IIS Issue 516

### PR DESCRIPTION
**Pull request by SmallMistake**
This pull request fulfills the three requests outlined in this issue:
https://github.com/DNNCommunity/DNNDocs/issues/516
1) Update the IIS Image
2) Include a note saying to use App Pool over Network Service
3) Include a note saying to not increase the number of workers on an App Pool

As for the first one about the IIS Image being out of date, it seems that the one currently up is almost identical to my IIS 10 structure. So. it might have already been updated. If it has been, feel free to revert it back to the old image if you want to keep the photoshopped elements the same as before.

### CODE REVIEW

**Review by valadas — CHANGES_REQUESTED** [review]
I _think_ the image here is probably a drag-and-drop into github, this will not work on the published site, an image needs to be actually part of the repository and then a relative link needs to be made to it like it was before this change

**Comment by SmallMistake** [comment]
Okay, let me change that on all the pull requests I have just submitted. I did not notice that there was a difference.

File: content/tutorials/server/setup/set-up-iis/index.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ links: ["[DNN Wiki: Setting up Your Module Development Environment](https://www.
 
 
 
-        ![IIS options to turn on](/images/scr-InstallIIS-6.png)
+        ![IISUpdated](https://user-images.githubusercontent.com/44900498/157516146-f7cb1f9d-058f-48d4-86bc-983eaa7897ab.png)
```
Comment (valadas) [inline content/tutorials/server/setup/set-up-iis/index.md:40]: Great but this link need to point to the image now :)

**Review by valadas — CHANGES_REQUESTED** [review]
Need to also update the link to point to the right image

File: content/tutorials/server/setup/set-up-iis/index.md (hunk 1 above)
Comment (SmallMistake) [reply content/tutorials/server/setup/set-up-iis/index.md]: Okay, I fixed the problem I was having due to saving the image in the wrong file. That should be all of them, but I'll double check just to be sure.

**Comment by SmallMistake** [comment]
Wait, now I have to fix that problem of the Localization branch being merged in. I'll fix that in a sec.

**Comment by SmallMistake** [comment]
Everything has gone haywire on this branch due to it getting merged with the others and then it got worse locally. I'll just close the pull request and start fresh after the others have gone through.

**Pull request closed without merging**